<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="filterLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAppendLogs"/>
   <addaction name="actionLoad_Config"/>
   <addaction name="actionCaseSensitiveFilter"/>
   <addaction name="actionRegExpFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAppendLogs">
   <property name="text">
    <string>Append Logs</string>
   </property>
  </action>
  <action name="actionLoad_Config">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
  <action name="actionCaseSensitiveFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Case Sensitive Filter</string>
   </property>
  </action>
  <action name="actionRegExpFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter by regular expretion</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
